packages/react-dom/src/__tests__/ReactDOMOption-test.js

```diff
@@ -41,9 +41,7 @@ describe('ReactDOMOption', () => {
     expect(() => {
       node = ReactTestUtils.renderIntoDocument(el);
     }).toWarnDev(
-      '<div> cannot appear as a child of <option>.\n' +
-        '    in div (at **)\n' +
-        '    in option (at **)',
+      '<div> cannot appear as a child of <option>.\n' + '    in option (at **)',
```

Comment:
This is a behavior change and I'm not sure how to avoid this with this solution.

Reply:
I think this is okay. Particularly since the problem is still identified in the message (even if the nest doesn't show up).

Reply:
@aweary or @gaearon Do you anticipate any problems with changing this warning text?